vmdb/app/models/ems_openstack.rb

```diff
@@ -123,4 +123,8 @@ def vm_delete_evm_snapshot(vm, image_id)
     $log.debug err.backtrace.join("\n") if $log.debug?
     raise
   end
+
+  def self.event_monitor_class
```

Comment:
@jrafanie hm, so it's definitely caused by this, but I cannot try now, could you? I will try this on a different setup. That would mean the test is actually correct and there was a bug before, when this method was missing.